src/IJulia.jl

```diff
@@ -264,7 +264,7 @@ push_postexecute_hook(f::Function) = push!(postexecute_hooks, f)
 Remove a function `f()` from the list of functions to
 execute after executing any notebook cell.
 """
-pop_postexecute_hook(f::Function) = splice!(postexecute_hooks, findfirst(x -> x == postexecute_hooks, f))
+pop_postexecute_hook(f::Function) = @compat splice!(postexecute_hooks, findfirst(equalto(f), postexecute_hooks))
```

Comment:
You don't need the `@compat` macro just for a function like `equalto` — this function is provided normally by Compat.  `@compat` is only required to rewrite syntax.

Reply:
Cool, I removed `@compat` then.